tests/system/providers/amazon/aws/example_redshift.py

```diff
@@ -111,7 +111,9 @@ def delete_security_group(sec_group_id: str, sec_group_name: str):
     conn_id_name = f"{env_id}-conn-id"
     sg_name = f"{env_id}-sg"
 
-    set_up_sg = setup_security_group(sec_group_name=sg_name, ip_permissions=[IP_PERMISSION])
+    get_vpc_id = get_default_vpc_id()
```

Comment:
I don't need to add this to the `chain()` below because there is an explicit xcom dependency with the next task (we pass the result of this task to it), so it'll be scheduled accordingly.
I can do it for readability if you feel it's better though.